app_data/pages.yaml

```diff
@@ -0,0 +1,11 @@
+- url: /communities/horizen/info # The pid_value i.e. (horizen) to be replaced according to the respective branded community
+  title: HorizonZen - Information 
```

Comment:
question: does it need quotes?

Reply:
no, even the template doesn't, I'll remove it.